src/extensions/default/CodeFolding/main.js

```diff
@@ -393,9 +394,9 @@ define(function (require, exports, module) {
             return prefs.getSetting("alwaysUseIndentFold");
         }, indentFold);
 
-        CodeMirror.registerHelper("fold", "django", CodeMirror.helpers.fold.brace);
-        CodeMirror.registerHelper("fold", "tornado", CodeMirror.helpers.fold.brace);
```

Comment:
No - it also turns out that the CodeMirror.helpers.fold.brace is undefined at the point this helpers are registered. Consequently no point in having them here. Also I don't think brackets actually support these modes so they are better addressed in an extension that provides support for the mode